docs/policy/container-release-policy.md

```diff
@@ -0,0 +1,46 @@
+Container Release Policy
+========================
+
+12 March 2019
+
+Container images are an increasingly popular tool for shortening the software development lifecycle, allowing for speedy
+deployment of new software versions or additional instances of a service.
+Select services in the OSG Software Stack will be distrbuted as container images to support VOs and sites that are
+interested in this model.
+
+This document contains policy information for container images distributed by the OSG Software Team.
+
+Contents and Sources
+--------------------
+
+Similar to our existing RPM infrastructure, container image sources, build logs, and artifacts will be stored in
+publicly available repositories (e.g. GitHub, DockerHub) for collaboration and traceability.
+Additionally, container images distributed by the OSG Software team will be based off of the latest version of a 
+[supported platform](https://opensciencegrid.org/docs/release/supported_platforms/) with software installed from OS,
+EPEL, and [osg-development](/policy/software-releases#yum-repositories) Yum repositories.
```

Comment:
Why development?

Reply:
One of the drivers for releasing container images in the first place is to shorten the software development lifecycle so osg-development (osg-minefield in actuality but that's an internal detail) is the logical place to pull software from.

Reply:
(We discussed this in person and agreed that we should only be pulling selected packages out of minefield, not the whole software stack)